dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ScriptRestRepository.java

```diff
@@ -80,7 +81,11 @@ public ScriptRest findOne(Context context, String name) {
     @Override
     @PreAuthorize("hasAuthority('ADMIN')")
     public Page<ScriptRest> findAll(Context context, Pageable pageable) {
-        List<ScriptConfiguration> scriptConfigurations = scriptService.getScriptConfigurations(context);
+        List<ScriptConfiguration> scriptConfigurations =
+            scriptService.getScriptConfigurations(context)
+                         .stream()
+                         .sorted(Comparator.comparing(ScriptConfiguration::getName))
+                         .collect(Collectors.toList());
```

Comment:
@corrad82-4s :  Couldn't we just add this sorting into `getScriptConfigurations()`, so that we only have to `stream()` this list once?   It looks like we could just move this into the `stream()` here: https://github.com/DSpace/DSpace/blob/main/dspace-api/src/main/java/org/dspace/scripts/ScriptServiceImpl.java#L38-L39

If I'm wrong though, please let me know.  Maybe I'm missing something, but I think this would have better performance if we moved the sorting into `ScriptService.getScriptConfigurations()`

Reply:
@tdonohue I see and agree with your point. I have pushed required change